### DISCUSSION TITLE
documentation

### DISCUSSION THREAD

**Discussion by ShinyFabio** [jokergoo/ComplexHeatmap#898] [General]
Hi jokergoo,
I like very much the documentation for this package ( https://jokergoo.github.io/ComplexHeatmap-reference/book/ ).
How can I do that? I used pkgdown like I think you, but I don't have that sidebar with all the chapters. I have something more similar to this: https://jokergoo.github.io/ComplexHeatmap/.

Did you use another package to build the book? 
Thank you in advance.

**Comment by jokergoo** [comment]
Hi, I use bookdown to build the book. You can see the tutorials at https://bookdown.org/.

**Comment by ShinyFabio** [comment]
Great! Thank you!
